salt/vault/firewall.sls

```diff
@@ -0,0 +1,25 @@
+
+{% from "vault/map.jinja" import vault with context %}
+
+firewalld_vault_service:
+  firewalld.service:
+    - name: vault
+    - ports:
+      - 8200/tcp
+      - 8201/tcp
+
+firewalld_vault_zone:
+  firewalld.present:
+    - name: vaultzone
+    - services:
+      - vault
+    - sources:
+{%- for mac, properties in salt.grains.get('meta-data:network:interfaces:macs', {}).items() %}
+  {%- if properties['device-number'] | int == 0 %}
+    {%- for cidr in properties['vpc-ipv4-cidr-blocks'].split('\n') %}
+      - {{ cidr }}
```

Comment:
So, I kind of shortcutted things before, but the proper place for this logic would be in map.jinja... Maybe add a `cidrs` key to the `vault` dictionary, then change the reference here to simply be:

```
    - sources: {{ vault.inbound_cidrs }}
```

Reply:
also added input variable for `inbound_cidrs`. Passing it in as grains from `appscripts.sh`